ichnaea/async/settings.py

```diff
@@ -86,6 +83,4 @@
 CELERY_TASK_SERIALIZER = 'internal_json'
 
 # cleanup
-del REDIS_URI
 del TASK_QUEUES
```

Comment:
I'm not sure why there is a variable cleanup section at all. Is it for memory management? Or should these be underscore names like `_TASK_QUEUES` so Python treats them as internal variables?

Reply:
It's for namespace management. That way no one can import `REDIS_URI` from this module.

You suggest prefixing it with `_` so then there's the indication it's private to the module to other developers. That'd work. We could also do `__` (two underscores) and Python will mange the actual name which makes it harder to import.

I think if it were me, I'd go the "we could do it, but meh" route and just remove the cleanup section altogether.

Reply:
I'm going with the "we don't need this" and removing it.